src/APP/pages/community/sections/JoinSYTForm.jsx

```diff
@@ -9,6 +9,13 @@ function JoinSYTForm({ closeModal }) {
   const [loading, setLoading] = useState(false);
   const [err, setErr] = useState("");
 
+  const [showInvite, setShowInvite] = useState(false);
+
+  function handleClick() {
+    closeModal();
+    window.open("https://discord.com/invite/xSH9eAgBJm", "_blank");
```

Comment:
Try to avoid using `window` object unless it's a last resort. It can cause memory leaks when its events aren't handled properly. 

Therefore, to mitigate this risk, React has a `React Router` to handle some of these events. In this case, you could use [`useNavigate` hook](https://reactrouter.com/en/main/hooks/use-navigate) to navigate the user to a new tab with the Discord invite url.